### ISSUE TITLE
Got error when run two instances of adminserver with chart in k8s

### ISSUE THREAD

**Issue by ywk253100** [goharbor/harbor#5937]
All pods start up succefully eventually and Harbor works well, but got some error logs in adminserver.

Status of pods:
```
harbor-for-ci-harbor-adminserver-6c7d64f64c-6vnzl     1/1       Running   3          4h
harbor-for-ci-harbor-adminserver-6c7d64f64c-ctbp6     1/1       Running   2          4h
harbor-for-ci-harbor-chartmuseum-6799855d6f-785xs     1/1       Running   0          4h
harbor-for-ci-harbor-chartmuseum-6799855d6f-l6ccj     1/1       Running   0          4h
harbor-for-ci-harbor-clair-7485599ff8-4922p           1/1       Running   2          4h
harbor-for-ci-harbor-clair-7485599ff8-n89t9           1/1       Running   2          4h
harbor-for-ci-harbor-core-7b9cbdb6b7-8hvbj            1/1       Running   2          4h
harbor-for-ci-harbor-core-7b9cbdb6b7-kzjgz            1/1       Running   2          4h
harbor-for-ci-harbor-database-0                       1/1       Running   0          4h
harbor-for-ci-harbor-jobservice-6d9969d676-f9n9z      1/1       Running   2          4h
harbor-for-ci-harbor-jobservice-6d9969d676-k78rl      1/1       Running   2          4h
harbor-for-ci-harbor-notary-server-f75b46767-52hzg    1/1       Running   0          4h
harbor-for-ci-harbor-notary-server-f75b46767-hf25l    1/1       Running   0          4h
harbor-for-ci-harbor-notary-signer-6c5c6cfb4d-dwsff   1/1       Running   0          4h
harbor-for-ci-harbor-notary-signer-6c5c6cfb4d-q995c   1/1       Running   0          4h
harbor-for-ci-harbor-portal-7f5dc957c6-2s7nm          1/1       Running   0          4h
harbor-for-ci-harbor-portal-7f5dc957c6-rqnlq          1/1       Running   0          4h
harbor-for-ci-harbor-registry-7f466c6677-7757r        2/2       Running   0          4h
harbor-for-ci-harbor-registry-7f466c6677-bsnbn        2/2       Running   0          4h
harbor-for-ci-redis-master-0                          1/1       Running   0          4h
```

Logs of adminserver:
```
2018-09-20T21:55:50Z [INFO] initializing system configurations...
2018-09-20T21:55:50Z [INFO] Registering database: type-PostgreSQL host-harbor-for-ci-harbor-database port-5432 databse-registry sslmode-"disable"
2018-09-20T21:55:50Z [INFO] Register database completed
2018-09-20T21:55:50Z [INFO] Upgrading schema for pgsql ...
2018-09-20T21:55:55Z [INFO] the path of json configuration storage: /etc/adminserver/config/config.json
2018-09-20T21:55:55Z [INFO] the path of key used by key provider: /etc/adminserver/key
2018-09-20T21:55:55Z [FATAL] [main.go:46]: failed to initialize the system: pq: duplicate key value violates unique constraint "properties_k_key"
```

Logs of database:
```
here1
hehe2
The files belonging to this database system will be owned by user "postgres".
This user must also own the server process.

The database cluster will be initialized with locales
  COLLATE:  en_US.UTF-8
  CTYPE:    en_US.UTF-8
  MESSAGES: C
  MONETARY: C
  NUMERIC:  C
  TIME:     C
The default text search configuration will be set to "english".

Data page checksums are disabled.

fixing permissions on existing directory /var/lib/postgresql/data ... ok
creating subdirectories ... ok
selecting default max_connections ... 100
selecting default shared_buffers ... 128MB
selecting dynamic shared memory implementation ... posix
creating configuration files ... ok
running bootstrap script ... ok
performing post-bootstrap initialization ... ok
syncing data to disk ... ok

Success. You can now start the database server using:

    pg_ctl -D /var/lib/postgresql/data -l logfile start


WARNING: enabling "trust" authentication for local connections
You can change this by editing pg_hba.conf or using the option -A, or
--auth-local and --auth-host, the next time you run initdb.
hehe3
root
waiting for server to start....LOG:  database system was shut down at 2018-09-20 21:51:48 UTC
LOG:  MultiXact member wraparound protections are now enabled
LOG:  autovacuum launcher started
LOG:  database system is ready to accept connections
 done
server started
ALTER ROLE


/entrypoint.sh: running /docker-entrypoint-initdb.d/initial-notaryserver.sql
CREATE DATABASE
CREATE ROLE
ALTER ROLE
GRANT


/entrypoint.sh: running /docker-entrypoint-initdb.d/initial-notarysigner.sql
CREATE DATABASE
CREATE ROLE
ALTER ROLE
GRANT


/entrypoint.sh: running /docker-entrypoint-initdb.d/initial-registry.sql
CREATE DATABASE


waiting for server to shut down...LOG:  received fast shutdown request
LOG:  aborting any active transactions
LOG:  autovacuum launcher shutting down
.LOG:  shutting down
LOG:  database system is shut down
 done
server stopped

PostgreSQL init process complete; ready for start up.

FATAL:  the database system is starting up
LOG:  database system was shut down at 2018-09-20 21:53:06 UTC
LOG:  MultiXact member wraparound protections are now enabled
LOG:  database system is ready to accept connections
LOG:  autovacuum launcher started
ERROR:  duplicate key value violates unique constraint "pg_type_typname_nsp_index"
DETAIL:  Key (typname, typnamespace)=(schema_migrations, 2200) already exists.
STATEMENT:  CREATE TABLE IF NOT EXISTS schema_migrations (version bigint not null primary key);
ERROR:  duplicate key value violates unique constraint "schema_migrations_pkey"
DETAIL:  Key (version)=(1) already exists.
STATEMENT:  INSERT INTO schema_migrations (version) VALUES ($1)
ERROR:  duplicate key value violates unique constraint "lock_name_key"
DETAIL:  Key (name)=(updater) already exists.
STATEMENT:  INSERT INTO Lock(name, owner, until) VALUES($1, $2, $3)
LOG:  incomplete startup packet
LOG:  incomplete startup packet
ERROR:  duplicate key value violates unique constraint "properties_k_key"
DETAIL:  Key (k)=(reload_key) already exists.
STATEMENT:  INSERT INTO "properties" ("k", "v") VALUES ($1, $2) RETURNING "id"
LOG:  incomplete startup packet
LOG:  incomplete startup packet
LOG:  incomplete startup packet
LOG:  incomplete startup packet
LOG:  incomplete startup packet
LOG:  incomplete startup packet
LOG:  incomplete startup packet
ERROR:  duplicate key value violates unique constraint "lock_name_key"
DETAIL:  Key (name)=(updater) already exists.
STATEMENT:  INSERT INTO Lock(name, owner, until) VALUES($1, $2, $3)
ERROR:  duplicate key value violates unique constraint "lock_name_key"
DETAIL:  Key (name)=(updater) already exists.
STATEMENT:  INSERT INTO Lock(name, owner, until) VALUES($1, $2, $3)
ERROR:  duplicate key value violates unique constraint "lock_name_key"
DETAIL:  Key (name)=(updater) already exists.
STATEMENT:  INSERT INTO Lock(name, owner, until) VALUES($1, $2, $3)
ERROR:  duplicate key value violates unique constraint "lock_name_key"
DETAIL:  Key (name)=(updater) already exists.
STATEMENT:  INSERT INTO Lock(name, owner, until) VALUES($1, $2, $3)
ERROR:  duplicate key value violates unique constraint "lock_name_key"
DETAIL:  Key (name)=(updater) already exists.
STATEMENT:  INSERT INTO Lock(name, owner, until) VALUES($1, $2, $3)
ERROR:  duplicate key value violates unique constraint "lock_name_key"
DETAIL:  Key (name)=(updater) already exists.
STATEMENT:  INSERT INTO Lock(name, owner, until) VALUES($1, $2, $3)
ERROR:  duplicate key value violates unique constraint "lock_name_key"
DETAIL:  Key (name)=(updater) already exists.
STATEMENT:  INSERT INTO Lock(name, owner, until) VALUES($1, $2, $3)
ERROR:  duplicate key value violates unique constraint "lock_name_key"
DETAIL:  Key (name)=(updater) already exists.
STATEMENT:  INSERT INTO Lock(name, owner, until) VALUES($1, $2, $3)
ERROR:  duplicate key value violates unique constraint "lock_name_key"
DETAIL:  Key (name)=(updater) already exists.
STATEMENT:  INSERT INTO Lock(name, owner, until) VALUES($1, $2, $3)
ERROR:  duplicate key value violates unique constraint "lock_name_key"
DETAIL:  Key (name)=(updater) already exists.
STATEMENT:  INSERT INTO Lock(name, owner, until) VALUES($1, $2, $3)
WARNING:  corrupted statistics file "pg_stat_tmp/global.stat"
```

**Comment by reasonerjt** [comment]
As for this error:
`duplicate key value violates unique constraint "schema_migrations_pkey"` in DB log
This can be fixed by creating `schema_migration` table by default in the db container.

I'll fix this issue and do some testing to try to reproduce others.

Seems we may need to make sure when initializing the properties the instance of admin server can ignore certain errors

**Comment by reasonerjt** [comment]
Fixed via #6083 by ignoring the dup key error while initializing the configuration items in DB.
